merge_duplicate_contacts/wizard/merge_contact_view.xml

```diff
@@ -1002,19 +1002,32 @@
         <field name="model">base.partner.merge.automatic.wizard</field>
         <field name="inherit_id" ref="base.base_partner_merge_automatic_wizard_form" />
         <field name="arch" type="xml">
-            <field name='exclude_contact' position="after">
+            <field name="exclude_contact" position="attributes">
+                <attribute name="invisible">1</attribute>
+            </field>
+            <field name="exclude_journal_item" position="attributes">
+                <attribute name="invisible">1</attribute>
+            </field>
+            <field name="exclude_contact" position="after">
                 <field name="associate_contact" />
                 <field name="contact_not_being_customer" />
                 <field name="without_sales_orders" />
+                <field name="filter_domain_email" />
             </field>
             <field name="group_by_parent_id" position="after">
-                <field name="group_by_domain_email" />
             </field>
```

Comment:
You don't need the positioning marker anymore. Please also delete the tags around